packages/entity-browser/src/components/SearchForm/SearchForm.js

```diff
@@ -13,14 +13,16 @@ const SearchForm = props => {
     props.setSearchInput()
   }
 
-  const msg = id => {
-    return props.intl.formatMessage({
-      id
-    })
+  const msg = id => (props.intl.formatMessage({id}))
+
+  const isHidden = name => {
+    const field = props.preselectedSearchFields.find(f => f.id === name)
+    return field !== undefined && field.hidden
```

Comment:
I think `return field && field.hidden` is generally better in cases like this, because everything would still work if line 19 was changed so that it returns `null` instead of `undefined`. We don't care if the field is `undefined` or `null`, we just want to know if "it's there".